yarn.lock

```diff
@@ -8973,7 +8973,7 @@ dns-equal@^1.0.0:
   resolved "https://registry.yarnpkg.com/dns-equal/-/dns-equal-1.0.0.tgz#b39e7f1da6eb0a75ba9c17324b34753c47e0654d"
   integrity sha1-s55/HabrCnW6nBcySzR1PEfgZU0=
 
-dns-packet@^1.3.4:
+dns-packet@^1.3.1, dns-packet@^1.3.4:
```

Comment:
This entry was automatically adjusted after regenerating the lockfile since other deps still point to `^1.3.1`.

Reply:
I manually did the update. we may need to update dns-packet again soon. We are 4 major release behind.

Reply:
These will just reappear the next time the lockfile is changed. The way it works is each entry lists its own dependencies, and those dependencies in turn are each given their own entry. Each entry has a range that it matches, and different ranges can stack for an entry.

So in our case, `mutlicast-dns` depends on `dns-packet` at 1.3.1 or higher. This will always be true and the lockfile will always reset itself to reflect that:

```
multicast-dns@^6.0.1:
  version "6.2.3"
  ...
  dependencies:
    dns-packet "^1.3.1"
    ...
```
The lockfile will then have an entry for `dns-packet`, and since we have it set specifically to `^1.3.4` in the `resolutions` list, it will make sure that all versions of `dns-packet` resolve to 1.3.4 by stacking the versions onto the same entry:

```
dns-packet@^1.3.1, dns-packet@^1.3.4:
  version "1.3.4"
  ...
```

This means that anything that requires `dns-packet` with either `^1.3.1` or `^1.3.4` will point to this specific entry, which is set to `1.3.4`. But in order to ensure that `multicast-dns` gets its proper version, it explicitly puts the version in the entry's list.